packages/node-sdk/example/app.ts

```diff
@@ -0,0 +1,73 @@
+import express from "express";
+import { bucket } from "./bucket";
+import { BucketClient } from "../src";
+import type { AppFlags } from "./flags";
+
+export const app = express();
+
+declare global {
+  namespace Express {
+    interface Locals {
+      bucketUser: BucketClient;
+    }
+  }
+}
+
+app.use(express.json());
+app.use((req, res, next) => {
+  const bucketUser = bucket
+    .withUser(req.headers["x-bucket-user-id"] as string, {
+      attributes: {
+        role: req.headers["x-bucket-is-admin"] ? "admin" : "user",
+      },
+    })
+    .withCompany(req.headers["x-bucket-company-id"] as string, {
+      attributes: {
+        betaUser: !!req.headers["x-bucket-company-beta-user"],
+      },
+    });
+
+  res.locals.bucketUser = bucketUser;
+  next();
+});
+
+const todos = ["Buy milk", "Walk the dog"];
+
+app.get("/", (_req, res) => {
+  res.locals.bucketUser.trackFeatureUsage("Front Page Viewed");
+  res.send("Hello World");
+});
+
+app.get("/todos", async (_req, res) => {
+  const flags = res.locals.bucketUser.getFlags<AppFlags>();
+
+  if (flags["show-todos"]) {
```

Comment:
this is actually wrong:
<img width="794" alt="Screenshot 2024-07-22 at 11 22 54" src="https://github.com/user-attachments/assets/7d22b12b-439a-4681-8691-f47bfcdc128b">

for the `getFlags` equivalent in the ReactSDK we went with a simple `Record<string, boolean>` after feedback from Lasse/Peter: https://github.com/bucketco/bucket-javascript-sdk/blob/main/packages/react-sdk/src/index.tsx#L286-L314

Since there's no `loading` state for this call, i think it makes sense here too. I initially made it more complicated, but that was a premature abstraction and it's not useful at the moment.